### PR TITLE
Streamlit Deployment for Project 2

### PR DESCRIPTION
**Pull request by Odeyiany2**
I made changes to the file I made a PR for previously. This new web.py file shows the workflow for this project which is to translate text if text input and then first transcribe audio to text and translate.

### CODE REVIEW

File: Project_2/web.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import streamlit as st 
+import pandas as pd
+from io import StringIO
```
Comment (Sammybams) [inline Project_2/web.py:3]:
```suggestion
```

File: Project_2/web.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    ('Yoruba', 'Igbo', 'Hausa'))
+
+dict_lang = {"Yoruba": "yo",
+               "Igbo": "ig",
+               "Hausa": "ha"}
+
+if option=='A text':
+    title = st.text_input('English Text', 'Enter your english text here?')
+    translation = translate(dict_lang[language], title)
+    st.write('Your translation from English to {language} is: ', translation)
```
Comment (Sammybams) [inline Project_2/web.py:22]:
```suggestion
    st.write(f'Your translation from English to {language} is: ', translation)
```

**Review by Sammybams — APPROVED** [review]
Thank you very much @Odeyiany2. I made some changes to your pull request by removing the redundant `from io import StringIO import`command and converting the line of code that will show the out and reference the language selected to f-string so it can actually get the value. See the difference below. 

Screenshot 2023-10-29 at 17 02 51


Screenshot 2023-10-29 at 17 00 44

**Comment by Odeyiany2** [comment]
Ohh that honestly skipped my sight probably because I wasn't using my glasses while I was making the correction😅 Thank you very much @Sammybams

**Pull request merged**